### PR TITLE
DeltaChat is a Gmail app alternative

### PR DESCRIPTION
**Pull request by monperrus**

### CODE REVIEW

**Comment by tycrek** [comment]
I'm not sure I understand what this is. Is it just a standard mail client that looks like a messaging app? A lot of their "features" are just standard email features.

**Comment by monperrus** [comment]
Yes it's a full-fledged email client, hence this PR to add it.

File: yaml/degoogle.yml [hunk 1]
```diff
@@ -1143,6 +1143,12 @@ mobile applications:
       eyes: null
       text: |
         Available on F-Droid.
+    - name: DeltaChat
+      url: https://delta.chat/
+      fdroid: com.b44t.messenger
+      eyes: null
+      text: |
+        Supports autocrypt
```
Comment (tycrek) [inline yaml/degoogle.yml:1151]: Text should be a bit more descriptive

**Review by tycrek — CHANGES_REQUESTED** [review]
Only thing before I merge is the `text` should be a bit more descriptive (I know others have bad descriptions, this is something that will be worked on)

File: yaml/degoogle.yml (hunk 1 above)
Comment (monperrus) [reply yaml/degoogle.yml]: done @tycrek

**Review by tycrek — APPROVED** [review]
Thanks @monperrus!

**Pull request merged**